editor/editor_data.h

```diff
@@ -224,33 +224,43 @@ class EditorData {
 	EditorData();
 };
 
+/**
+ * Stores and provides access to the nodes currently selected in the editor.
+ *
+ * This provides a central location for storing "selected" nodes, as a selection can be triggered from multiple places,
+ * such as the SceneTreeDock or a main screen editor plugin (e.g. CanvasItemEditor).
+*/
```

Comment:
Fix Javadoc-style comment end alignment:

```suggestion
 */
```